content/en/docs/operations/faq.md

```
@@ -243,3 +243,77 @@ Moved to the [Bundles reference]({{% ref "docs/operations/bundles#how-to-overwri
 ### How to disable some components from bundle
 
 Moved to the [Bundles reference]({{% ref "docs/operations/bundles#how-to-disable-some-components-from-bundle" %}}).
+
+### Public Network Kubernetes Deployment
+
+In certain scenarios, a Kubernetes cluster is deployed where:
+  - All nodes (master and workers) use public IP addresses.
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 0; Actual: 2

(MD007, ul-indent)